### ISSUE TITLE
r/api_management_api: support for tags

### ISSUE THREAD

**Issue by jbaisley** [hashicorp/terraform-provider-azurerm#9200]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

We want to be able to add API tags and then apply these tags to the applicable APIs.

![api-tags](https://user-images.githubusercontent.com/29050257/102511352-ebec8700-4045-11eb-9cde-27b9a02ebde2.png)

Or only apply tags to the APIs, since there can be many.

![image](https://user-images.githubusercontent.com/29050257/98382727-cc0a8200-2008-11eb-81bc-6e1951b29e90.png)

### New or Affected Resource(s)

* azurerm_2.32.0

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by Priya-Raut** [comment]
@jbaisley You might want to consider updating screenshot in the description.
It has been renamed to '**API tags**':
![image](https://user-images.githubusercontent.com/13897222/102510430-9f3c9680-40ad-11eb-9d24-6e7e13661d37.png)

**Comment by jbaisley** [comment]
This has been resolved since 2.32, closing issue.
